pelorus-operator/config/rbac/role.yaml

```diff
@@ -1,106 +1,15 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
+  labels:
+    app.kubernetes.io/name: clusterrole
+    app.kubernetes.io/instance: manager-role
+    app.kubernetes.io/component: rbac
+    app.kubernetes.io/created-by: pelorus-operator
+    app.kubernetes.io/part-of: pelorus-operator
+    app.kubernetes.io/managed-by: kustomize
   name: manager-role
 rules:
-##
```

Comment:
is this removal of permissions ok :question:

Reply:
TBH I don't know as this is generated by the operator-sdk, so we need to test that operator to see.